src/PyCall.jl

```diff
@@ -110,7 +124,7 @@ function pydecref(o::PyObject)
 end
 
 function pyincref_(o::Union{PyPtr,PyObject})
-    ccall((@pysym :Py_IncRef), Cvoid, (PyPtr,), o)
+    @pyccall(:Py_IncRef, Cvoid, (PyPtr,), o)
```

Comment:
I *guess* `Py_DecRef` can call `__del__` so it should probably be protected but I wonder if `Py_IncRef` can ever call any Python functions.